current_ranges.test.py

```diff
@@ -0,0 +1,19 @@
+import math
+import unittest
+import current_ranges
+
+
+class CurrentRangesTest(unittest.TestCase):
```

Comment:
Clean attempt. Suggestion: name the tests with input and outputs in the name. E.g., test_empty_input_returns_error.
Go ahead, pass these tests, write a test to see if get_current_ranges gives the correct ranges. Let it fail (because it's currently returning True).
Then write the implementation to pass it.